src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,62 @@
+package duke;
+import duke.exception.*;
+
+import java.io.IOException;
+
+public class Duke {
+    public static void main(String[] args) {
+        Init.showWelcomeMessage();
+        try {
+            Save.loadData();
+        } catch(IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+        while(true) {
+            Parser.input = Parser.getInput();
+            Parser.taskList = Save.taskList;   //load data if there is any
```

Comment:
Please maintain spacing between the comment specifier (//) and the first word of your comment. You might want to also consider adding comments in new lines to improve code readability.